samples/dfu/ab/README.rst

```diff
@@ -0,0 +1,105 @@
+.. _ab_sample:
+
+A/B with MCUboot Sample Application
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample demonstrates how to configure the application, so it will be updated using the "A/B" method using MCUBoot as well as a sample way to perform a device healthcheck before confirming the image after the update.
+
+You can update the sample using SMP protocol with UART or Bluetooth® Low Energy.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+You need the nRF Device Manager app for SUIT update over Bluetooth Low Energy:
+
+* `nRF Device Manager mobile app for Android`_
+* `nRF Device Manager mobile app for iOS`_
+
+
+Overview
+********
+
+This sample demonstrates firmware update using "A/B" method.
+This method allows to have two copies of the application in the NVM memory.
+It is possible to switch between these copies, without the need to perform swap what significantly reduces time of device's unavailability during the update.
+Switch between images can be triggered by application or for example HW button.
+
+This sample application implements a Simple Management Protocol (SMP) server.
+SMP is a basic transfer encoding for use with the MCUmgr management protocol.
+For more information about MCUmgr and SMP, please see :ref:`device_mgmt`.
+
+This sample application supports the following MCUmgr transports by default:
+
+    * Bluetooth
+    * Serial (UART)
+
+
+Configuration
+*************
+
+The :kconfig:option:`CONFIG_N_BLINKS` Kconfig option sets the number of times the LED corresponding to the currently active slot blinks (LED0 for slot A, LED1 for slot B).
```

Comment:
Edits are done to follow the template https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/templates/sample_README.html#configuration_options.
```suggestion
Configuration options
=====================

Check and configure the following configuration option for the sample:

.. _CONFIG_N_BLINKS:

CONFIG_N_BLINKS - The number of blinks.
   This configuration option sets the number of times the LED corresponding to the currently active slot blinks (LED0 for slot A, LED1 for slot B).
```